Utility.cpp

```diff
@@ -2,14 +2,45 @@
 #include <iostream>
 #include <string>
 #include <ctype.h>
+#include <vector>
+#include <fstream>
 
 //Method for checking if a given input is a number or not
-bool Utility::isNumber(std::string input) {
+bool Utility::isNumber(std::string input) 
+{
     if (input.size() == 0) return false;
-    for (char c : input) {
-        if (!isdigit(c)) {
+    for (char c : input) 
```

Comment:
Itt meg további felesleges másolások minden karakterre. `for (const char& c : input)`